### DISCUSSION TITLE
Why can't I create an AppWindow with WindowsAppSDK 1.3 and WinUI3

### DISCUSSION THREAD

**Discussion by LZD121** [microsoft/microsoft-ui-xaml#8519] [Q&A]
I saw in release note that WindowsApp SDK 1.3 have already support  AppWindow, why can't I use the API?
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/59716053/6860dc82-e774-474c-bae8-39a8018ffd00)

**Comment by DarranRowe** [comment]
One difficulty here is that it is difficult to know what you want as an answer.
The simple answer is that the Windows App SDK doesn't implement this behaviour. This, however, may not be the kind of answer that you are looking for.
The Windows App SDK/WinUI 3 doesn't implement everything available in the UWP, and I think currently the only way to get content into the Xaml host in WinUI 3 is to use Microsoft.UI.Xaml.Window.Content.
